.github/workflows/publish-apple-darwin-wasm.yml

```diff
@@ -87,3 +87,13 @@ jobs:
           asset_name: nargo-${{ matrix.target }}.tar.gz
           overwrite: true
           tag: ${{ inputs.noir-ref || 'nightly' }} # This will fail if noir-ref is not a tag (e.g. testing a branch)
+
+      - uses: JasonEtco/create-an-issue@v2
+        if: ${{ failure() }}

```

Comment:
Just making a note that this needs to be restricted to scheduled calls so that we don't raise issues and ping people when running the workflows in a PR on this repo (other than in this one for testing).